test/client/mongodb_service.test.js

```diff
@@ -297,8 +297,8 @@ describe('Client API executing mongodb service functions', () => {
     ]);
 
     expect(results).toMatchObject([
-      { _id: 'bar', total: 2 },
```

Comment:
interesting - do we know why this order changed?

Reply:
Not sure to be honest, I haven't touched the client much to be honest. Maybe @makesitgo might have an idea ?

Reply:
according to docs, [`$group` does not do any ordering](https://docs.mongodb.com/manual/reference/operator/aggregation/group/#pipe._S_group) ... seems like this was always fluky slash we were just getting lucky.

i'd recommend maybe just adding `{ $sort: { _id: 1 } }` to the pipeline, instead of reversing the order of the expected match object's fields

Reply:
Thank you, will do!